src/AdTypes/McXml.php

```diff
@@ -46,18 +46,9 @@ public function __construct($partner_id, $provider)
         $this->VIDEO_URL = '';
         $this->initializeContact();
         $this->contactBody->removeAttribute('PHONESALESRESERVATION');
-        $this->contactBody->removeChild($this->contactFaxBody);
         $this->contactBody->removeChild($this->contactURLBody);
     }
 
-    public function setContactFax($fax)
-    {
-    }
-
-    public function setContactUrl($url)
```

Comment:
Maybe we should keep this one? We don't want it to be possible to set contact url if its not present in the Dtd